app/views/lead_providers/your_schools/index.html.erb

```diff
@@ -1,9 +1,22 @@
-<%= content_for :before_content, govuk_breadcrumbs(breadcrumbs: {
-  "Dashboard" => dashboard_path,
-  "Your Schools" => nil
-}) %>
+<% content_for :title, "Your schools" %>
 
-<%= render ::SchoolInformation::YourSchools::HeadingComponent.new %>
+<% content_for :nav_bar do %>
+  <%= render PrimaryNavComponent.new do |component| %>
+    <%= component.nav_item(path: dashboard_path) do %>
+      Overview
+    <% end %>
+    <%= component.nav_item(path: lead_providers_your_schools_path) do %>
+      Schools
+    <% end %>
+  <% end %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Your schools</h1>
+    <%= govuk_link_to 'Confirm more schools', start_lead_providers_report_schools_path, button: true %>
```

Comment:
We don't have a linter for that, but we try to keep a convention of using double quotes in views.